src/style.css

```
@@ -1,3 +1,96 @@
 body {
   margin: 0;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 50px;
+}
+
+.header__logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+}
+
+.header__image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.header__nav {
+  display: flex;
+}
+
+.header__list {
+  list-style: none;
+  margin: 0;
+  display: flex;
+  padding: 0;
+}
+
+.header__item {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  height: 60px;
+  position: relative;
+}
+
+.header__item:not(:first-child) {
+  margin-left: 10px;
+}
+
+.header__item:not(:last-child) {
+  margin-right: 10px;
+}
+
+.header__link {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  text-align: center;
+  text-transform: uppercase;
+  transition: all 0.3s ease 0s;
+}
+
+.header__link:hover {
+  color: var(--hover-color);
+}
+
+.header__item.is-active > .header__link {
+  color: var(--hover-color);
+}
+
+.header__item.is-active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  width: 37px;
+  height: 4px;
+  background-color: var(--hover-color);
+  bottom: 0;
+  border-radius: 100px;
+}
+
+.header__link[data-qa='hover']:hover {
```

Use only classes selector
```suggestion
.header__link:hover {
```